### PR TITLE
Add Kuneco section to Community page 

### PR DESCRIPTION
**Pull request by michellewong793**
image

Add Calendar embed: 
image

### CODE REVIEW

File: src/community/CommunityPage.tsx [hunk 1]
```diff
@@ -29,6 +33,7 @@ export function CommunityPage() {
           <ArticleData title={t("articles.title")} />
           <Contribute />
           <EcoFund />
+          {props.sections ? items : <></>}
```
Comment (aaronmgdr) [inline src/community/CommunityPage.tsx:36]: only thing is it should be above Contribute component

File: src/utils/contentful.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { Page as SideBarEntry } from "src/experience/common/Sidebar"
 import { Props as BlurbProps } from "src/contentful/grid2-cells/Blurb"
 import { BTN } from "src/shared/Button.3"
 import { fetchCached, MINUTE } from "src/../server/cache"
+import { StringValueNode } from "graphql"
```
Comment (aaronmgdr) [inline src/utils/contentful.ts:9]: this looks like an accidental import

File: src/community/CommunityPage.tsx [hunk 3]
```diff
@@ -27,6 +31,7 @@ export function CommunityPage() {
           <CoverArea />
           <Tenets />
           <ArticleData title={t("articles.title")} />
+          {props.sections ? items : <></>}
```
Comment (aaronmgdr) [inline src/community/CommunityPage.tsx:34]: could save a line and the ternary by just placing line  19 here ` props.sections?.map(pageSwitch)`

Comment (michellewong793) [reply src/community/CommunityPage.tsx:34]: sure! thanks

File: pages/community.tsx [hunk 4]
```diff
@@ -1,11 +1,19 @@
...
 import { serverSideTranslations } from "next-i18next/serverSideTranslations"
 import { NameSpaces } from "src/i18n"
+import { getPageBySlug } from "src/utils/contentful"
+
+export async function getServerSideProps({ locale }) {
+  const page = await getPageBySlug("community", { locale: "en-US" }, true)
+
+  if (!page) {
+    return { notFound: true }
+  }
```
Comment (aaronmgdr) [inline pages/community.tsx:11]: given that this page has lots of non contentful based sections. I think we would be better off still showing the page just without the contentful part.

**Review by aaronmgdr — APPROVED** [review]

**Pull request merged**